plugins/explore/src/components/DomainCard/DomainCardGrid.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { DomainEntity } from '@backstage/catalog-model';
+import { Grid } from '@material-ui/core';
+import React from 'react';
+import { DomainCard } from '.';
+
+type DomainCardGridProps = {
+  entities: DomainEntity[];
```

Comment:
At this point i started thinking about what it MEANS to import a non-versioned kind :) I think later on, we'll have to start actually settling this and deciding who has to normalize the schemas - the writer side (so that the catalog only ever STORES one version of schema), or the reader side

Reply:
(never mind this now ... but at some point we need to talk about it)